python/overlay_widget/shotgun_model_overlay_widget.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+# 
+# CONFIDENTIAL AND PROPRIETARY
+# 
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your 
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Overlay widget specifically designed to work with a shotgun model (from the 
+tk-framework-shotgunutils framework).  Multiple overlay widgets can be
+easily created and connected to the same shotgun model.
+"""
+
+from sgtk.platform.qt import QtCore, QtGui
+
+# load resources
+from .ui import resources_rc
+from .shotgun_overlay_widget import ShotgunOverlayWidget
+
+class ShotgunModelOverlayWidget(ShotgunOverlayWidget):
```

Comment:
GOF say *prefer Composition over inheritance*. Just double checking here since this doesn't seem clear cut. Should this maybe be a HAS-A rather than a IS-A?